### DISCUSSION TITLE
Should we have protocol fees?

### DISCUSSION THREAD

**Discussion by andreivladbrg** [sablier-labs/flow#24] [General]
Since we have removed the protocol fees for the next Lockup release, should we abandon the idea to implement them in this project?

Ref (last point in TODOs): https://github.com/sablier-labs/v2-open-ended/blob/main/README.md#todos

@PaulRBerg @smol-ninja

**Comment by smol-ninja** [comment]
I think yes. As arguments made in https://github.com/sablier-labs/company-discussions/discussions/2, we can implement it back in the future after we have good traction with the v2-open-ended or when we think it is time to switch on the protocol fee. And by not implementing it right now, in comparison, our contracts will be more gas-efficient on ethereum.

**Comment by PaulRBerg** [comment]
Correct, we will _not_ implement them in OpenEnded.

**Comment by PaulRBerg** [comment]
**Update**: we changed our mind about this and decided to implement protocol fees:

- https://github.com/sablier-labs/company-discussions/discussions/67
- https://github.com/sablier-labs/flow/issues/213
